paket.dependencies

```diff
@@ -2,11 +2,14 @@ version 5.198.0
 source https://api.nuget.org/v3/index.json
 frameworks: net5.0, netstandard2.0
 
-nuget FSharp.Core 4.3.4
+nuget FSharp.Core 4.7.0
 nuget System.Net.Http
-nuget Newtonsoft.Json
+nuget System.Net.Http.Json
+nuget System.Text.Json
+nuget FSharp.SystemTextJson
```

Comment:
Why do we need 3 explicitly referenced packages here?
```
 nuget System.Net.Http.Json
 nuget System.Text.Json
 nuget FSharp.SystemTextJson
```
can we reference only `FSharp.SystemTextJson`?

Reply:
I thought that we will use JSON extensions to `HttpClient` and `HttpResponseMessage`

Reply:
I suppose we can